frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,35 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
+                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" required focus-if>
                             <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
                             <label for="old_password" data-error="wrong" data-success="right">Old Password*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password">{{changepwd.wrnMsg.old_password}}</div>
+                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password" ng-messages="changepwdForm.old_password.$error">{{changepwd.wrnMsg.old_password}}
+                                <p ng-message="required">Old password is required</p>
+                            </div>
                         </div>
 
                         <div class="input-field align-left">
-                            <input id="new_password1" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" focus-if>
+                            <input id="new_password1" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" ng-minlength="8" required focus-if>
```

Comment:
Add name attributes here too.
FYI. `ng-if` is used to focus on input. and you can't activate multiple input fields at a time.
remove `ng-if`. No need to add here since it is already focusing the old password input.